### PR TITLE
Simplifying the structure and UI

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: src/client/styles/card.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ *
+ */
+ .card {
+   box-shadow: 0 4px 5px 0 rgba(0,0,0,.14), 0 1px 10px 0 rgba(0,0,0,.12), 0 2px 4px -1px rgba(0,0,0,.2);
```
Comment (samdutton) [inline src/client/styles/card.scss]: Nit: let's add CSS rules in alphabetical order. Best practice and avoids linter complaints.

File: src/client/styles/footer.scss [hunk 2]
```diff
@@ -16,19 +16,16 @@
  *  limitations under the License
  *
  */
- 
+
 footer {
   max-width: 944px;
```
Comment (samdutton) [inline src/client/styles/footer.scss]: I wonder if some 'magic numbers' like this should be variables?

File: src/client/styles/card.scss (hunk 1 above)
Comment (devnook) [reply src/client/styles/card.scss]: I'm Chaotic Evil. You?

lawful-good-ordering-css-properties-by-logical-grouping-neutral-good-28163412

File: src/client/styles/footer.scss (hunk 2 above)
Comment (devnook) [reply src/client/styles/footer.scss]: Sure! Let's do it in a separate PR though.

**Comment by devnook** [comment]
Thanks Sam! @rowan-m, let's take a look at this code again at a later stage together.

**Pull request merged**